channels/views_test.py

```diff
@@ -395,21 +398,13 @@ def test_get_post_not_found(client, logged_in_profile, use_betamax, praw_setting
 
 
 @pytest.mark.parametrize("missing_user", [True, False])
-def test_list_posts(
-        client,
-        use_betamax,
-        praw_settings,
-        private_channel_and_contributor,
-        reddit_factories,
-        missing_user
-):
+def test_list_posts(client, missing_user, private_channel_and_contributor, reddit_factories):
     """List posts in a channel"""
     channel, user = private_channel_and_contributor
-
-    first_post = reddit_factories.text_post('my post', user, channel=channel)
-    second_post = reddit_factories.text_post('my 2nd post', user, channel=channel)
-
-    time.sleep(3)
+    posts = list(reversed([
```

Comment:
What's the `reversed` here for?

Reply:
It's because the api returns the posts in the reverse order that they're made, so this is a inline form of:

```python
post1 = reddit_factories.post(...)
post2 = reddit_factories.post(...)

posts = [post2, post1]
```

If you feel that's clearer, I can change it, but they're functionally equivalent.